Plugins/DICOMParametricMapPlugin.py

```diff
@@ -1,164 +1,165 @@
-import os

```

Comment:
I think it would be good to call this directory `DICOMPlugins` instead of `Plugins`